src/Pact/Types/Exp.hs

```diff
@@ -25,7 +25,7 @@ module Pact.Types.Exp
  (
    Literal(..),
    _LString,_LInteger,_LDecimal,_LBool,_LTime,
-   simpleISO8601,formatLTime,
+   simpleISO8601,formatLTime,renderLiteral,
```

Comment:
We're not using this so should probably ditch it, esp. since it conflicts with our `Pretty` stuff

Reply:
Ahh shoot i totally forgot about that. My bad, reverting.